lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
@nickcharlton I realized as I was typing it that probably only Americans called it that, and that shamefully I knew no other name for it 😓 . `twenty_four_hour_time` just doesn't roll off the keys.